service/src/test/java/greencity/service/ubs/ViolationServiceImplTest.java

```diff
@@ -82,19 +83,10 @@ void deleteViolationThrowsException() {
         assertThrows(UserNotFoundException.class, () -> violationService.deleteViolation(1L, "abc"));
     }
 
-    @Test
-    void deleteViolationFromOrderResponsesNotFoundWhenNoViolationInOrder() {
```

Comment:
Why did you remove this test?
Toy should mock  `employeeRepository.findByUuid(uuid)` here